server/plugin/subscriptions.go

```diff
@@ -17,6 +17,7 @@ const (
 	SubscriptionsKey     = "subscriptions"
 	excludeOrgMemberFlag = "exclude-org-member"
 	renderStyleFlag      = "render-style"
+	FeaturesFlag         = "features"
```

Comment:
@TeodorPt What do you think about namespacing the constants like so:

```suggestion
	flagExcludeOrgMember = "exclude-org-member"
	flagRenderStyle      = "render-style"
	flagFeatures         = "features"
```